src/cls/IPM/Storage/Module.cls

```diff
@@ -593,6 +593,7 @@ Method GetDefaultParameters(Output pParams)
 
             // Convert "."-delimited pieces to subscripts in pParams
             set tPieces = $listfromstring(tDefault.Name,".")
+            kill tValue
```

Comment:
Why is this needed if tValue is set right under it? Or could this be multidimensional?